Client/Cards/index.ts

```diff
@@ -36,14 +37,25 @@ export class Cards extends List<
 	static create(connection: Connection): Cards {
 		return new Cards(connection)
 	}
-	async getAllCard(page?: number, pageSize?: number) {
-		let path
-		if (page || pageSize)
-			path = `v2/cards?page=${page ?? 0}&size=${pageSize ?? 20}`
-		else
-			path = `v2/cards`
-		const result = await this.connection.get<model.CardResponseV2[]>(path)
-		return result
+	async getAllCardsPaginated(
+		previous?: Paginated<model.CardResponseV2>,
+		page?: number,
+		size?: number,
+		sort = "issueDate,desc"
```

Comment:
can you do createdOn,desc instead

issue date sucks, its only per day, createdOn shows you the actual minutes etc